tests/loaders/query/test_QueryLoaderStrategyMzxml.py

```diff
@@ -0,0 +1,81 @@
+from csodiaq.loaders.query.QueryLoaderStrategy import QueryLoaderStrategy
+from csodiaq.loaders.query.QueryLoaderStrategyMzxml import QueryLoaderStrategyMzxml
+import pytest
+import os
+from expectedPooledQueryPeaks import (
+    expectedOutputPoolSingleScan,
+    expectedOutputPoolMultipleScans,
+)
+
+
+def get_parent_dir():
+    return os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
Minor: This basically just does, "go up three directories"; `../../../` will do the same.  
The function should basically be restricted to this file; use a leading underscore `_get_parent_dir` to signal "private" functions. Python doesn't enforce private/public methods, but it's convention to notify devs/users that it's not intended for use in other contexts.

Reply:
I'll be resolving this in a future commit - making a function that gets the test_file directory path that the test files can import.